README.md

```diff
@@ -38,21 +35,18 @@ that are too broadly scoped to lead to agreement, so be as specific as you can.
 
 **Example:** *The Barricade can be circumvented too easily with wall jumps.*
 
-## Proposal
+## Idea, that solves a problem
```

Comment:
1. No comma needed.
2. Maybe change to "Solution Proposal" or add "(Proposal)" behind this so that we still have a keyword for our labels.